libglusterfs/src/inode.c

```diff
@@ -562,12 +567,14 @@ __inode_ref(inode_t *inode, bool is_invalidate)
             inode->in_invalidate_list = false;
             inode->table->invalidate_size--;
         } else {
-            inode->table->lru_size--;
+            if (inode->in_lru_list)
```

Comment:
If this is false, it's a bug that needs to be fixed. Have you found any case where this happens ?